docs/_templates/layout.html

```diff
@@ -1,10 +1,9 @@
-{% extends "!layout.html" %}
+{% extends "pydata_sphinx_theme/layout.html" %}
```

Comment:
Why did this need to change? [According to the docs](https://www.sphinx-doc.org/en/master/development/templating.html#jinja-sphinx-templating-primer) the `!` should make things pull from the underlying theme.

Reply:
Good call, shouldn't be needed